### PR TITLE
chapter 05_0 translated and ready for review

### PR DESCRIPTION
**Pull request by icculp**
Translated and ready for review by native speaker

### CODE REVIEW

File: es/05_0_Controlling_Bitcoin_Transactions.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Capitulo Cinco: Controlar Las Transacciones de Bitcoin 
+
+Enviar una transacción no siempre es el final de la historia. Al usar los protocolos RBF (reemplazo por tarifa) y CPFP (el niño paga por el padre), un desarrollador puede continuar controlando la transacción después de que se haya enviado, para mejorar la eficiencia o recuperar transacciones que se atascan. Estos métodos comenzarán a destacar el cerdadero poder de Bitcoin.
```
Comment (javiervargas) [inline es/05_0_Controlling_Bitcoin_Transactions.md:3]: el verdadero poder de

**Review by javiervargas — CHANGES_REQUESTED** [review]
Fix typos

**Comment by shannona** [comment]
Folks,

Great to see this all coming along! It looks like the system of one person writing and another reviewing, leading to revisions is working well, and the way that you're keeping each section to a PR is keeping it all nice and clean.

I am assuming this is still in process. If (and when) it's ready to go, I just need either the reviewer to actually check off that it's ready, or either of you to post a comment saying it's ready for a merge.

Thanks again! Very exciting to see this translation going forward.

**Comment by javiervargas** [comment]
Still we need to check some internals links,   although translation  it's ok.

**Comment by shannona** [comment]
Great. Just let me know when it's ready to merge.

**Comment by shannona** [comment]
Hey folks, just wanted to check in and see how things are going on the Spanish translation. No rush, I just want to make sure that *I'm* not holding things up. When each PR has been reviewed and revised, the best way to alert me is to either request a review from me or to tag me in a comment telling me you're ready to merge, and I'll then do so.

**Review by javiervargas — APPROVED** [review]

**Pull request merged**